tests/Type/WebMozartAssert/data/type.php

```diff
@@ -215,10 +215,10 @@ public function isInstanceOfAny($a, $b, $c, $d, $e, $f, $g): void
 		assertType('mixed', $d);
 
 		Assert::isInstanceOfAny($e, [17]);
-		assertType('mixed', $e);
+		assertType('object', $e);
 
 		Assert::isInstanceOfAny($f, [17, self::class]);
-		assertType('PHPStan\Type\WebMozartAssert\TypeTest', $f);
+		assertType('object', $f);
```

Comment:
This is the only type that seems to have gotten _less_ specific, this is because we now always fall back to `object` when we can't find any class names, so we end up with `object|PHPStan\Type\WebMozartAssert\TypeTest` which is then further normalized to `object`.

I don't know how big of a deal this is as this code would fail in runtime (`17` not being a valid class name)